sklearn/feature_selection/rfe.py

```diff
@@ -313,6 +313,11 @@ class RFECV(RFE, MetaEstimatorMixin):
         Defaults to 1 core. If `n_jobs=-1`, then number of jobs is set
         to number of cores.
 
+    n_features_to_select : int, default=1
```

Comment:
I believe this does not precisely address #6564 since setting a fixed number of final features (`n_features_to_select`) is different from setting a minimum (`min_n_features`). I have posted #8812 in an attempt to address this difference.